docs/application/dotnet/guides/nui/styling-controls-with-JSON.md

```diff
@@ -230,6 +230,36 @@ You can implement transitions in two ways:
     ]
     ```
 
+## Apply JSON style
+
+To apply Style on runtime, use StyleManager as the following code:
+
+```
+using Tizen.NUI.Components;
+using Tizen.NUI.BaseComponents;
+
+//Load json theme
+Tizen.NUI.StyleManager.Get().ApplyTheme(DirectoryInfo.Resource + "/styles/theme.json");
+
+//Create button object
+Button testButton = new Button();
+testButton.SetStyleName("CustomButton");
```

Comment:
By the way, I have a question at this line.

From where did we set the styleName as "CustomButton"?

Reply:
CustomButton is a style name defined in JSON file. Maybe I should change it to testButton.SetStyleName("PushButton") which is defined in this documentation or testButton.SetStyleName(<Style Name>). What do you think about it?

Reply:
Thanks for the comment, @mihashco 
I agree it would be better to change the name of JSON style to other one.

As "PushButton" is described through the page, how about using "PushButton"?

Reply:
Sure, It was changed.